src/HealthChecks.Publisher.Seq/HealthChecks.Publisher.Seq.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>$(NetStandardTargetVersion)</TargetFramework>
+        <PackageLicenseUrl>$(PackageLicenseUrl)</PackageLicenseUrl>
+        <PackageProjectUrl>$(PackageProjectUrl)</PackageProjectUrl>
+        <PackageTags>HealthCheck;Publisher;Seq</PackageTags>
+        <Description>HealthChecks.Publisher.Seq is the health check publisher for Seq.</Description>
+        <Version>$(HealthCheckPublisherPrometheus)</Version>
+        <RepositoryUrl>$(RepositoryUrl)</RepositoryUrl>
+        <Company>$(Company)</Company>
+        <Authors>$(Authors)</Authors>
+        <PackageId>AspNetCore.HealthChecks.Publisher.Seq</PackageId>
```

Comment:
You need to create a new variable version for this project on  dependencies.props, the version for prometheus and this project can't be the same